main.rb

```diff
@@ -52,24 +62,31 @@ def run
     end
   end
 
-  def display_options
-    puts 'Please choose an option by entering a number:'
-    puts '1 - List all books'
-    puts '2 - List all music albums'
-    puts '3 - List of games'
-    puts '4 - List all genres'
-    puts '5 - List all labels'
-    puts '6 - List all authors'
-    puts '7 - Add a book'
-    puts '8 - Add a music album'
-    puts '9 - Add a game'
-    puts '10 - Remove a music album'
-    puts '11 - Exit'
+  def display_options(selected_option)
+    options = [
+      '1️⃣   List all books 📚',
+      '2️⃣   List all music albums 🎶 📀',
+      '3️⃣   List of games 🎮',
+      '4️⃣   List all genres  🎵 🎶',
+      '5️⃣   List all labels 🏷️  📚',
+      '6️⃣   List all authors ✍️  📖',
+      '7️⃣   Add a book 📚 ✏️',
+      '8️⃣   Add a music album 🎶 📀 🎵',
+      '9️⃣   Add a game 🎮 🕹️',
+      '🔟  Exit 🚪 👋'
+    ]
```

Comment:
Hello @batoolfatima2135, I think you've accidentally deleted the remove album option when you're resolving merge conflicts. Kindly add the option again. 😁